networking/network_observability/installing-operators.adoc

```diff
@@ -14,11 +14,20 @@ The Loki Operator integrates a gateway that implements multi-tenancy & authentic
 The Loki Operator can also be used for xref:../../logging/cluster-logging-loki.adoc#cluster-logging-loki[Logging with the LokiStack]. The Network Observability Operator requires a dedicated LokiStack separate from Logging.
 ====
 
+include::modules/network-observability-without-loki.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
+* For more information about exporting flow data to Kafka for third party processing consumption, see xref:../../networking/network_observability/configuring-operator.adoc#network-observability-enriched-flows-kafka_network_observability[Export enriched network flow data].
+
+//IMPORTANT: this file network-observability-enriched-flows-kafka.adoc changes to etwork-observability-enriched-flows.adoc with PR#63324, so at merge time, this will need updating. For now if I change it, my build will be broken.
```

Comment:
Would a follow-up Jira be a better solution than a note in the docs that could get skipped if you were to be moved onto another project or priorities changes, and so on?